src/main/java/hudson/plugins/git/extensions/impl/GitLFSPull.java

```diff
@@ -76,6 +96,13 @@ public String toString() {
 
     @Extension
     public static class DescriptorImpl extends GitSCMExtensionDescriptor {
+        public ListBoxModel doFillCredentialsIdItems(@AncestorInPath Item project,
+                                                     @QueryParameter String url,
+                                                     @QueryParameter String credentialsId) {
+            return new UserRemoteConfig.DescriptorImpl().doFillCredentialsIdItems(
```

Comment:
I am not sure if this intended to be reused like this. @MarkEWaite what's your opinion?